OnionBrowser/Browsing/BrowsingViewController+Tabs.swift

```diff
@@ -84,12 +84,12 @@ UICollectionViewDropDelegate, TabCellDelegate {
 		return cell
 	}
 
-
+	
 	// MARK: UICollectionViewDelegate
 
 	func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
 		if let cell = collectionView.cellForItem(at: indexPath) as? TabCell {
-			currentTab = tabs.first { $0 == cell.container.subviews.first }
```

Comment:
That was there for a reason. Now you produce a warning: "Value 'cell' was defined but never used..." Why do we need to switch this?